### ISSUE TITLE
Wrong version of Lazarus listed

### ISSUE THREAD

**Issue by thecoalitionofchaos** [cheat-engine/cheat-engine#2299]
In the readme it says you need 2.2.2 to build the code, but you need 2.0.10 - 2.2.2 gives an error: 
'frmAAEditPrefsUnit.pas(50,16) Error: Identifier not found "TSynEditorOptions"'
Really not sure how this kind of mistake happens lmao. Just tried it out myself and it worked perfectly on 2.0.10

**Comment by fade2gray** [comment]
Yep, the build instructions in the README changed recently, but doesn't inform you that you need to use 2.2.2 for the master source and 2.0.10 for the release source.

**Comment by cheat-engine** [comment]
the 2.0.10 source has a readme.md file as well. Read that one as it does state the correct version

**Comment by SuperFoxy8888** [comment]
> Yep, the build instructions in the README changed recently, but doesn't inform you that you need to use 2.2.2 for the master source and 2.0.10 for the release source.

So you need 2.0.10 for the compilation to work?

**Comment by cheat-engine** [comment]
it depends on which version of the sourcecode you have.

Read the readme.md of the sourcecode you have downloaded
